src/main/java/org/skriptlang/skript/bukkit/recipes/elements/ExprRecipeResult.java

```diff
@@ -0,0 +1,93 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.Recipe;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.recipes.MutableRecipe;
+import org.skriptlang.skript.bukkit.recipes.CreateRecipeEvent;
+
+@Name("Recipe Result")
+@Description("The result item for a recipe.")
+@Examples({
+	"set {_recipe} to a new shaped recipe with the key \"my_recipe\":",
+		"\tset the recipe ingredients of 1st row to diamond, air and diamond",
+		"\tset the recipe result to diamond sword named \"Chosen One\""
+})
+@Since("INSERT VERSION")
+public class ExprRecipeResult extends PropertyExpression<Recipe, ItemStack> {
+
+	static {
+		Skript.registerExpression(ExprRecipeResult.class, ItemStack.class, ExpressionType.PROPERTY,
+			"[the] recipe result[ing] [item] [of %recipes%]",
```

Comment:
```suggestion
			"[the] result[ing] recipe [item] [of %recipes%]",
```

Reply:
I'm not a fan of this, because then that allows for
`set {_get} to the resulting recipe of {_recipe}`
which doesnt add up when it's supposed to get the item.